### DISCUSSION TITLE
Issue: High RAM Usage and Laptop Reboot During Prompt Execution

### DISCUSSION THREAD

**Discussion by Alameen1433** [lllyasviel/Fooocus#2458] [Q&A] [answered]
After downloading Fooocus from the official repository and following the installation procedure correctly, running run.bat opens the browser and launches Fooocus successfully. However, upon giving a prompt, the RAM usage spikes to 100%, causing the laptop to reboot.Also I've installed recommended graphics drivers to ensure compatibility. Any suggestions for troubleshooting or potential fixes would be greatly appreciated.
System Information:
Processor: AMD Ryzen 7
System RAM: 16GB
GPU: Nvidia GeForce RTX 3050 TI 4GB VRAM

**Comment by poor7** [comment, marked as answer]
There is probably not enough memory swap file size on disk. Make 30-40 GB.
https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#system-swap
